### DISCUSSION TITLE
Best deployment strategy

### DISCUSSION THREAD

**Discussion by kjy5** [miguelgrinberg/python-socketio#1257] [General]
There are so many options for [deployment](https://python-socketio.readthedocs.io/en/stable/server.html#deployment-strategies). Is there a "best", "most modern", "most performant", etc. option? In what scenario should I be choosing one strategy over another?

Thanks!

**Comment by miguelgrinberg** [comment]
There is no best option. How you deploy the application doesn't really affect the application in a big way. The choice will depend on your own preferences more than anything.

This is equivalent to asking if nginx or apache are better for a static website. Some people will prefer nginx, some other apache, but at the end of the day it doesn't really matter, both serve static files so the user will have the same experience.

**Reply by kjy5** [reply]
Ah, understood. Thanks for the reply!
